src/sofe.js

```diff
@@ -12,61 +12,75 @@ export function applyMiddleware(...middleware) {
 	setMiddleWare(chain);
 }
 
-export { locate, fetch, isOverride };
+export { locate, fetch, isOverride, getServiceUrl, InvalidServiceName };
 
-export { getServiceName } from './utils.js';
+export { getServiceName } from "./utils.js";
 
 export function getManifest(url) {
-	if (typeof url !== 'string') {
+	if (typeof url !== "string") {
 		throw new Error(`sofe getManifest API must be called with a url string`);
 	}
 
 	return new Promise((resolve, reject) => {
 		// Ensure that the manifest for this url has been retrieved
-		_getManifest({manifestUrl: url})
-		.then(() => {
-			/* We don't want the merged combination of all of the chained manifests,
+		_getManifest({ manifestUrl: url })
```

Comment:
Not a big deal, but you could just return the `getManifest` promise to avoid having to call `.catch` everywhere